downstream-sample/downstream_sample/settings.py

```diff
@@ -0,0 +1,50 @@
+from contextlib import contextmanager
+from typing import ContextManager
+import os
+
+import pika
+from pika.adapters.blocking_connection import BlockingChannel
+
+
+DELIVERY_MODE_PERSISTENT = 2
+
+HOST = os.environ.get('RABBITMQ_HOST', 'localhost')
+PORT = os.environ.get('RABBITMQ_PORT', 5672)
+USER = os.environ.get('RABBITMQ_USER', 'guest')
+PASSWORD = os.environ.get('RABBITMQ_PASSWORD', 'guest')
+
+ARTICLE_EXCHANGE_NAME = 'articles'
+DOWNSTREAM_EXCHANGE_NAME = 'downstream-sample'
+DOWNSTREAM_QUEUE_NAME = 'downstream-sample'
+
+CREDENTIALS = pika.PlainCredentials(USER, PASSWORD)
+PARAMS = pika.ConnectionParameters(host=HOST, credentials=CREDENTIALS)
```

Comment:
Could we use `URLParameters` to simplify the environment variables (down to one).

Reply:
For reference, this was done in [#32](https://github.com/libero/walking-skeleton/pull/32) [here](https://github.com/libero/walking-skeleton/blob/5d3e0d65dcbc75e00d9f61cba619902a06321d6f/article-store/events/conf.py#L8).